src/EditorFeatures/Core/Implementation/Classification/SyntacticClassificationTaggerProvider.cs

```diff
@@ -25,15 +25,15 @@ namespace Microsoft.CodeAnalysis.Editor.Implementation.Classification
     internal partial class SyntacticClassificationTaggerProvider : ForegroundThreadAffinitizedObject, ITaggerProvider
     {
         private readonly IAsynchronousOperationListener _listener;
-        private readonly ClassificationTypeMap _typeMap;
+        private readonly SyntacticClassificationTypeMap _typeMap;
 
         private readonly ConditionalWeakTable<ITextBuffer, TagComputer> _tagComputers = new();
 
         [ImportingConstructor]
         [SuppressMessage("RoslynDiagnosticsReliability", "RS0033:Importing constructor should be [Obsolete]", Justification = "Used in test code: https://github.com/dotnet/roslyn/issues/42814")]
         public SyntacticClassificationTaggerProvider(
             IThreadingContext threadingContext,
-            ClassificationTypeMap typeMap,
+            SyntacticClassificationTypeMap typeMap,
```

Comment:
we also use a type map in the semantic tagger, does that need to be changed to use the semantic classification layer? https://sourceroslyn.io/#Microsoft.CodeAnalysis.EditorFeatures/Implementation/Classification/SemanticClassificationViewTaggerProvider.cs,49

Reply:
Hmm, I believe the semantic tagger shouldn't be affected since the `ClassificationLayer` enum is only utilized by the editor in LSP scenarios. LSP's semantic tagger automatically assigns the semantic classifications we send back to the client as `ClassificationLayer.Semantic`, so all we need to do is make sure our Roslyn syntactic tagger, which activates in the client, is marked as `ClassificationLayer.Syntactic` so that the two taggers don't interfere with each other.

At least that's my understanding, @gundermanc can correct me if I'm wrong